ts/components/ui/HeaderSecondLevel.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from "react";
+import { View, StyleSheet, Platform } from "react-native";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+import { useNavigation } from "@react-navigation/native";
+import Animated, {
+  interpolate,
+  interpolateColor,
+  useAnimatedStyle
+} from "react-native-reanimated";
+import I18n from "i18n-js";
+import { IOColors, hexToRgba } from "../core/variables/IOColors";
+import { WithTestID } from "../../types/WithTestID";
+import {
+  IOStyles,
+  IOVisualCostants,
+  iconBtnSizeSmall
+} from "../core/variables/IOStyles";
+import { HSpacer } from "../core/spacer/Spacer";
+import { makeFontStyleObject } from "../../components/core/fonts";
+import type { IOSpacer } from "../core/variables/IOSpacing";
+import IconButton from "./IconButton";
+
+export type HeaderSecondLevel = WithTestID<{
+  scrollValues: ScrollValues;
+  title: string;
+  // Accepted components: IconButton
+  // Don't use any components other than this, please.
+  firstAction?: React.ReactNode;
+  secondAction?: React.ReactNode;
+  thirdAction?: React.ReactNode;
+}>;
+
+type ScrollValues = {
+  contentOffsetY: Animated.SharedValue<number>;
+  triggerOffset: number;
+};
+
+const HEADER_BG_COLOR: IOColors = "white";
+const borderColorDisabled = hexToRgba(IOColors["grey-100"], 0);
+const titleHorizontalMargin: IOSpacer = 16;
+
+const styles = StyleSheet.create({
+  headerInner: {
+    backgroundColor: IOColors[HEADER_BG_COLOR],
```

Comment:
We should add the possibility to have a totally transparent header, like we have in the `CameraView` component design.

Reply:
@mastro993 Just added a new `transparent` prop. Also updated the PR description to describe the related behavior.